configure.ac

```diff
@@ -2948,12 +2948,12 @@ main(void)
     stack_t ss;
     struct sigaction sa;
 
-    ss.ss_sp = malloc(SIGSTKSZ);
+    ss.ss_sp = malloc(16*1024);
     if (ss.ss_sp == NULL) {
 	fprintf(stderr, "cannot allocate memory for sigaltstack\n");
 	return EXIT_FAILURE;
     }
-    ss.ss_size = SIGSTKSZ;
+    ss.ss_size = 16*1024;
```

Comment:
I got this size from here, in `signal.c`:


https://github.com/ruby/ruby/blob/37c2cd3fa47c709570e22ec4dac723ca211f423a/signal.c#L565-L569